### PR TITLE
cmake: Make numpy a first-class dependency for Python

### PR DESCRIPTION
**Pull request by mbr0wn**
Yeah, I hate dependencies. But let's not kid ourselves, numpy was always a dependency of GNU Radio. Might as well treat it as such. Should then obviate #3280.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -50,6 +50,7 @@ set(GR_SWIG_MIN_VERSION "3.0.8")
 set(GR_CMAKE_MIN_VERSION "3.5.1")
 set(GR_MAKO_MIN_VERSION "0.4.2")
 set(GR_PYTHON_MIN_VERSION "3.6.5")
+set(GR_NUMPY_MIN_VERSION "1.13.0")
```
Comment (mbr0wn) [inline CMakeLists.txt]: Note: This is what 18.04 has.

Comment (mormj) [reply CMakeLists.txt]: Seems like for master this is a good floor

**Comment by nickoe** [comment]
@morm @mbr0wn  is this something we need for maint-3.8?

**Comment by michaelld** [comment]
> @morm @mbr0wn is this something we need for maint-3.8?

Yes, if not done already. Would be useful for maint-3.7 too if that's easily doable ...

**Pull request merged**